package/src/main.rs

```diff
@@ -204,45 +211,102 @@ fn maybe_build(args: &Args, build_context: &BuildContext) -> Result<Option<PathB
             check,
             tools_pex_mismatch_warn,
         } => {
```

Comment:
It seems like the four branches of `Commands::Test` (and `Commands::Scie`) could be unified into one branch which initialized the `tools_pex` or `scie_pants` if they weren't set? I know this was here before, but as each branch gets longer (in very similar ways) the (apparent?) duplication is more obvious.

Reply:
Done here: #194